helm/templates/primary/statefulset-primary-rs.yaml

```diff
@@ -103,6 +117,10 @@ spec:
               subPath: {{ .Values.persistence.subPath }}
             - name: config
               mountPath: /etc/mongodb
+            {{- if .Values.usePassword }}
```

Comment:
Can the init-scripts flag be named accordingly (instead of using the `usePassword` flag for `initi-scripts`?

Reply:
removed the flag for init scripts and moved it inside the configmap only for mongo-users.js